_documentation/playback/auto_rewind.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Automatic rewind
+layout: doc
+level: "2"
+group: "playback"
+icon: "fas fa-keyboard"
+---
+
+# Automatic rewind after starting an episode which has been paused
+
+When an episode has been paused and is played again AntennaPod will automatically rewind.
+
+How much is rewinded is not configurable and depends on how long an episode has been paused.
+
+Rewinds timing are currently :
```

Comment:
I added a few words about *why* we rewind (also, removed the line breaks). Let me know what you think :)

```suggestion
When an episode has been paused and is played again, AntennaPod will automatically rewind, so it is easier to continue following the show. How much is rewound is not configurable and depends on how long an episode has been paused. Rewind timing are currently:
```